pkg/serverless/daemon/routes.go

```diff
@@ -104,7 +104,17 @@ func (e *EndInvocation) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	errorMsg := r.Header.Get(invocationlifecycle.InvocationErrorMsgHeader)
+	if decodedMsg, err := base64.StdEncoding.DecodeString(errorMsg); err != nil {
+		log.Debug("Could not decode error message header")
```

Comment:
Too ambiguous, set it as something like "couldn't decode base 64, setting as is" or something similar